### PR TITLE
Merge branch '2.2' into merge

### PR DESCRIPTION
**Pull request by aRn0D**
**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added legacy Behats               | -
| Added acceptance tests            | -
| Added integration tests           | -
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

File: src/Pim/Component/Catalog/Validator/Constraints/ActivatedLocaleValidator.php [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    {
+        $this->localeRepository = $localeRepository;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($locale, Constraint $constraint)
+    {
+        if (!$locale instanceof LocaleInterface) {
```
Comment (rouliane) [inline src/Pim/Component/Catalog/Validator/Constraints/ActivatedLocaleValidator.php]: for my personnal culture, why not just add a typehinting to the $locale parameter ? Is it for compatibility ?

Comment (aRn0D) [reply src/Pim/Component/Catalog/Validator/Constraints/ActivatedLocaleValidator.php]: because of Symfony design

File: .ci/behat.community.yml [hunk 2]
```diff
@@ -0,0 +1,146 @@
+legacy:
```
Comment (damien-carcel) [inline .ci/behat.community.yml]: I think this file and the enterprise one are useless now. The CI use `behat.yml.dist` at the root of the project.

File: upgrades/schema/Version_2_2_20180201155905_add_phone_on_user.php [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<?php
```
Comment (damien-carcel) [inline upgrades/schema/Version_2_2_20180201155905_add_phone_on_user.php]: Shouldn't those two files be moved in `upgrades/schema/2.2/` ?

File: src/Pim/Bundle/CatalogBundle/Version.php [hunk 4]
```diff
@@ -12,10 +12,10 @@
 class Version
 {
     /** @staticvar string */
-    const VERSION = '2.3.0';
+    const VERSION = '2.2.0-ALPHA2';
```
Comment (damien-carcel) [inline src/Pim/Bundle/CatalogBundle/Version.php]: 3.0 ?

**Pull request merged**